com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -126,22 +114,82 @@ define([
               'is_optgroup': '0',
               'is_reserved': '0',
               'is_active': '1'
+            },
+            {
+              'id': '846',
+              'option_group_id': '231',
+              'label': 'HR',
+              'value': 'HR',
+              'name': 'HR',
+              'is_default': '0',
+              'weight': '2',
+              'is_optgroup': '0',
+              'is_reserved': '0',
+              'is_active': '1'
+            },
+            {
+              'id': '847',
+              'option_group_id': '102',
+              'label': 'Headquarters',
+              'value': 'Headquarters',
+              'name': 'Headquarters',
+              'is_default': '0',
+              'weight': '1',
+              'is_optgroup': '0',
+              'is_reserved': '0',
+              'is_active': '1'
+            },
+            {
+              'id': '1160',
+              'option_group_id': '107',
+              'label': 'East',
+              'value': 'East',
+              'name': 'East',
+              'is_default': '0',
+              'weight': '1',
+              'is_optgroup': '0',
+              'is_reserved': '0',
+              'is_active': '1'
             }
           ],
           'optionGroupData': {
-            'cost_centres': '124'
+            'cost_centres': '124',
+            'hrjc_department': '231',
+            'hrjc_level_type': '111',
+            'hrjc_location': '102',
+            'hrjc_region': '107'
           }
         }));
         initController();
         $rootScope.$digest();
       });
 
       it('builds the "CostCentreList" array of objects containing the "weight" property', function () {
-        expect(ctrl.CostCentreList.length).toBe(3);
+        expect(Object.keys(ctrl.CostCentreList).length).toBe(3);
         expect(ctrl.CostCentreList[0].weight).not.toBeNull();
         expect(ctrl.CostCentreList[1].weight).not.toBeNull();
         expect(ctrl.CostCentreList[2].weight).not.toBeNull();
       });
+
+      it('builds the "hrjc_level_type" array of objects containing the "value" property', function () {
+        expect(Object.keys(ctrl.LevelsData).length).toBe(1);
+        expect(ctrl.LevelsData.value).not.toBeNull();
```

Comment:
No need for testing if it is not null, the test above will fail if it is null anyway. Same for other test suites.